internal/definitions/definition_handler.go

```diff
@@ -143,20 +144,19 @@ func (dh *definitionHandlers) HandleDefinitionBroadcast(ctx context.Context, sta
 	case core.SystemTagDefineContractAPI:
 		return dh.handleContractAPIBroadcast(ctx, state, msg, data, tx)
 	default:
-		l.Warnf("Unknown SystemTag '%s' for definition ID '%s'", msg.Header.Tag, msg.Header.ID)
-		return HandlerResult{Action: ActionReject}, nil
+		return HandlerResult{Action: ActionReject}, fmt.Errorf("unknown system tag '%s' for definition ID '%s'", msg.Header.Tag, msg.Header.ID)
```

Comment:
Does the fact that this now returns an error affect logic in the caller of this function?

Reply:
No. The only caller is in `aggregator.go`, and in the case of `ActionReject` it simply logs the error.